quickwit/scripts/.ee.license_header.txt

```diff
@@ -0,0 +1,20 @@
+// The Quickwit Enterprise Edition (EE) license
+// Copyright (c) {\d+}-present Quickwit Inc.
+//
+// With regard to the Quickwit Software:
+//
+// This software and associated documentation files (the "Software") may only be
+// used in production, if you (and any entity that you represent) hold a valid
```

Comment:
I assume the production part should allow test use cases, but the wording is not clear, e.g. it can be understand as it can only be used in production and nowhere else.

Maybe we can remove the production part, and it can be covered by "Quickwit Enterprise license corresponding to your usage". We would need to emit access tokens anyway.

Reply:
This is the wording used in several other famous license. It should be ok.